features/filesystem/bd/MBRBlockDevice.cpp

```diff
@@ -67,22 +67,22 @@ static void tochs(uint32_t lba, uint8_t chs[3])
 // Partition after address are turned into absolute
 // addresses, assumes bd is initialized
 static int partition_absolute(
-        BlockDevice *bd, int part, uint8_t type,
-        bd_size_t offset, bd_size_t size)
+    BlockDevice *bd, int part, uint8_t type,
+    bd_size_t offset, bd_size_t size)
 {
     // Allocate smallest buffer necessary to write MBR
     uint32_t buffer_size = std::max<uint32_t>(bd->get_program_size(), sizeof(struct mbr_table));
     uint8_t *buffer = new uint8_t[buffer_size];
 
     // Check for existing MBR
-    int err = bd->read(buffer, 512-buffer_size, buffer_size);
+    int err = bd->read(buffer, 512 - buffer_size, buffer_size);
     if (err) {
         delete[] buffer;
         return err;
     }
 
-    struct mbr_table *table = reinterpret_cast<struct mbr_table*>(
-            &buffer[buffer_size - sizeof(struct mbr_table)]);
+    struct mbr_table *table = reinterpret_cast<struct mbr_table *>(
+                                  &buffer[buffer_size - sizeof(struct mbr_table)]);
```

Comment:
Why was this indention level chosen?

Reply:
This is the default behavior for the style:
(`min-conditional-indent=1`) and not defined (`indent-after-parens`)

With `--indent-after-parens` defined:
```
@@ -82,7 +82,7 @@ static int partition_absolute(
     }
 
     struct mbr_table *table = reinterpret_cast<struct mbr_table *>(
-                                  &buffer[buffer_size - sizeof(struct mbr_table)]);
+            &buffer[buffer_size - sizeof(struct mbr_table)]);
     if (table->signature[0] != 0x55 || table->signature[1] != 0xaa) {
         // Setup default values for MBR
         table->signature[0] = 0x55;
@@ -149,7 +149,7 @@ int MBRBlockDevice::partition(BlockDevice *bd, int part, uint8_t type, bd_addr_t
 }
 
 int MBRBlockDevice::partition(BlockDevice *bd, int part, uint8_t type,
-                              bd_addr_t start, bd_addr_t stop)
+    bd_addr_t start, bd_addr_t stop)
 {
     int err = bd->init();
     if (err) {
@@ -204,7 +204,7 @@ int MBRBlockDevice::init()
 
     // Check for valid table
     struct mbr_table *table = reinterpret_cast<struct mbr_table *>(
-                                  &buffer[buffer_size - sizeof(struct mbr_table)]);
+            &buffer[buffer_size - sizeof(struct mbr_table)]);
     if (table->signature[0] != 0x55 || table->signature[1] != 0xaa) {
         delete[] buffer;
         return BD_ERROR_INVALID_MBR;
```

Reply:
I pushed an update to see the option actually in place, see https://github.com/ARMmbed/mbed-os/pull/6014/commits/14ea87da959791cfa0395ca6f3faf192ca51d282